crates/ruma-events/src/message.rs

```diff
@@ -122,6 +124,15 @@ pub struct MessageEventContent {
         deserialize_with = "crate::room::message::relation_serde::deserialize_relation"
     )]
     pub relates_to: Option<Relation<MessageEventContentWithoutRelation>>,
+
+    /// [MSC4095](https://github.com/matrix-org/matrix-spec-proposals/pull/4095)-style bundled url previews
+    #[cfg(feature = "unstable-msc4095")]
+    #[serde(
+        rename(serialize = "com.beeper.linkpreviews"),
```

Comment:
Why is there the `serialize =`? I am pretty sure that it means it will fail to deserialize with the unstable prefix.